src/ProjectSystemTools/source.extension.vsixmanifest

```diff
@@ -9,13 +9,13 @@
     <Tags>project, csproj, vbproj</Tags>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0.27004.2002,)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.0,)" />
   </Installation>
   <Dependencies>
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
   </Dependencies>
   <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,17.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
```

Comment:
> <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0) [](start = 3, length = 84)

This means that the tool will no longer work for 15.9. There is no way to have two different version of an extension published. If we do this we need to now create two different extension one that is for dev15 and another one for dev16. We had this discussion for the Nexus work and we didn't land a decision

Reply:
I do think we should probably adopt a versioning policy where we do it by release rather than an extension that will work forever, so that we can take advantage of new APIs as they come along. @madskristensen, is there any guidance about how best to do this given that we've already released an extension? Is it possible to rename the current extension to something like Project System Tools 2017 and then introduce a Project System Tools 2019? Or should we create a Project System Tools 2017 and then restrict this to 2019?

Reply:
Putting the year in the title is a common practice and one I use myself. That's my recommendation

Reply:
OK, what I would suggest is that we create a 2017 branch and change the title to "Project System Tools 2017". Then change the GUID in the main branch and the title to "Project System Tools 2019" and integrate this. It will mean there won't be a pointer for people to move (since 2017 will actually support 2019), but we can manually point people if they need it.

Reply:
Shouldn't the extension gallery just show old versions of the extension to users of earlier VS versions? If that's not how it currently behaves, perhaps it's how it should behave. Having to ship different extensions seems klunky and breaks things like extension roaming across major releases.

Reply:
Following up. I've:

- Updated this PR to add "2019" to the extension name
- Created a `vs-2017` branch
- Filed PR #379 against that branch which:
  - Adds "2017" to the extension name
  - Creates a new GUID identity for that extension

Next steps:

- Fix the package versioning woes on this PR
- Publish the new 2017 extension on the marketplace

Reply:
@panopticoncentral 

> It will mean there won't be a pointer for people to move (since 2017 will actually support 2019), but we can manually point people if they need it.

Reading your comment again, I see this is the inverse of what I've submitted in these PRs. I've made a new identity for the 2017 extension.

@madskristensen do you know what happens when a newer version of an extension does not support the version of VS it's installed into? Does it just not upgrade? If so then I think the approach I've embarked on should be ok, and allow users of the existing extension to get the latest features without doing anything manual.